### PR TITLE
A11y: Fix some accessibility issues on the build error page

### PR DESCRIPTION
**Pull request by tomratcliffe**
**What is this feature?**

Fixes accessibility issues on the build-error page that shows when displaying the UI went _really_ wrong.

**Why do we need this feature?**
* We need accessible colour contrast for the link (it was currently blue against dark background, and surrounded by orange text)
* The page needed a basic section (WAVE a11y checker was complaining that [we had no page regions](https://www.w3.org/WAI/GL/wiki/Using_ARIA_landmarks_to_identify_regions_of_a_page)), so a `main` has been added
* There was no heading element. The main error message has been made into an `h1`
* The code-related parts of the text have been changed to `code` elements and styled in the same way as the other code text in the main Grafana app
* Spacing was added to the containing element so it doesn't push up against the edge of the screen when in mobile/smaller viewports

To get the error page to show, run `yarn start` but then delete the `app.{hash}.js` file from `public/build`

**Who is this feature for?**
Everyone who uses the UI!

Before:
error-before
Before, with errors reported by WAVE:
error-before-wave

After:
error-after
After, with (no) errors reported by WAVE
error-after-wave

### CODE REVIEW

File: public/views/index.html [hunk 1]
```diff
@@ -162,6 +163,21 @@
         display: block;
       }
 
+      .preloader--done .preloader__text--fail a {
+        color: white;
```
Comment (tskarhed) [inline public/views/index.html:167]: This causes issues if you've been using the light theme (unclear when it is set). The theme is assigned to the body, so just `.light-theme` and add another color
image

File: public/views/index.html [hunk 2]
```diff
@@ -176,25 +192,29 @@
...
-          <br />
-          5. Check if you are using a non-supported browser. For more information, refer to the list of
-          <a href="https://grafana.com/docs/grafana/latest/installation/requirements/#supported-web-browsers">
-            supported browsers</a
-          >.
-        </p>
+        <main>
+          <h1>If you're seeing this Grafana has failed to load its application files</h1>
+          <p>
+            1. This could be caused by your reverse proxy settings.
```
Comment (tskarhed) [inline public/views/index.html:198]: I am having trouble reading beyond point 1 with VoiceOver. Maybe we should make this into and `ol` instead?

File: public/views/index.html (hunk 1 above)
Comment (tskarhed) [reply public/views/index.html:167]: The orange doesn't pass the contrast check here either

File: public/views/index.html [hunk 3]
```diff
@@ -254,7 +278,7 @@
         var cssLink = document.createElement("link");
         cssLink.rel = 'stylesheet';
 
-        if (darkQuery.matches) {
+        if (false && darkQuery.matches) {
```
Comment (tskarhed) [inline public/views/index.html:281]:
```suggestion
        if (darkQuery.matches) {
```

Comment (tomratcliffe) [reply public/views/index.html:281]: duh, sorry was trying to force it and forgot I left it in, thank you 🙏

**Review by tskarhed — APPROVED** [review]

**Pull request merged**